src/pages/ContactsItem/ContactsItem.js

```diff
@@ -0,0 +1,25 @@
+// @flow strict
+
+import { connect } from 'react-redux'
+
+import {
+  type Props,
+  ContactsItemView,
+} from 'pages/ContactsItem/ContactsItemView'
+
+type OwnProps = {|
+  +contactId: ContactId,
+|}
+
+function mapStateToProps(state: AppState, { contactId }: OwnProps) {
```

Comment:
We don't need mapStateToProps, witch does nothing

Reply:
But it will needed in future